.travis.yml

```diff
@@ -47,14 +47,15 @@ install:
     - git clone --branch $LIBECL_VERSION --depth 1 https://github.com/equinor/libecl
     - bash ert/.build_install.sh libecl
     - bash ert/.build_install.sh libres
-    - export PYTHONPATH=$INSTALL_DIR/lib/python$TRAVIS_PYTHON_VERSION/dist-packages:$PYTHONPATH
-    - export PYTHONPATH=$INSTALL_DIR/lib/python$TRAVIS_PYTHON_VERSION/site-packages:$PYTHONPATH
+    - export PYTHONPATH=$PYTHONPATH:$INSTALL_DIR/lib/python$TRAVIS_PYTHON_VERSION/dist-packages
```

Comment:
Why?

Reply:
The issue I have is that libres has a hard dependency for configsuite `0.5.3` (or had - I've removed it now as it wasn't a real dependency). All my last 5-6 commits are just trying to find a way to update configsuite to `0.6`, and that doesn't seem to be as easy as I thought.

I could just use the master branch of libres instead of using the one in `.LIBRES_VERSION` from `ert`, but I wanted to see if I could find a more sane way, as this isn't really an edge case.